FlightJournal.Web/Translations/da/LC_MESSAGES/messages.po

```diff
@@ -1472,7 +1629,8 @@ msgid ""
 "Training data for this flight have been edited by someone else, please "
 "reload!"
 msgstr ""
-"Skolingsdata for denne flyvning er blevet ændret af en anden, du er desværre nødt til at genindlæse siden!"
+"Skolingsdata for denne flyvning er blevet ændret af en anden, du er desværre "
```

Comment:
JS dør hvis denne er splittet over to linier - jeg har håndrettet det hver gang, men vi skal nok have en mere robust løsning.

Reply:
Er fixet nu i PR #96